packages/salesforcedx-vscode-lwc/src/index.ts

```diff
@@ -64,16 +75,34 @@ export async function activate(context: ExtensionContext) {
       ]
     }
   };
-
   // Create the language client and start the client.
   const disposable = new LanguageClient(
     'lwcLanguageServer',
     'LWC Language Server',
     serverOptions,
     clientOptions
   ).start();
-
   // Push the disposable to the context's subscriptions so that the
   // client can be deactivated on extension deactivation
   context.subscriptions.push(disposable);
 }
+
+export function populateEslintSettingIfNecessary(
+  context: ExtensionContext,
+  config: WorkspaceConfiguration
+) {
+  const nodePath = config.get<string>(ESLINT_NODEPATH_CONFIG);
+
+  // User has not set one, use the eslint bundled with our extension
+  // or if it is from salesforcedx-vscode-lwc, update since the path looks like
+  // "eslint.nodePath": ".../.vscode/extensions/salesforce.salesforcedx-vscode-lwc-41.17.0/node_modules",
```

Comment:
Since the bundling includes the version number, we need to make sure to update this with each version. Rather than checking for the version number explicitly, we just update it each time.